src/indra_cogex/sources/processor.py

```diff
@@ -231,3 +271,10 @@ def validate_relations(relations):
         except Exception as e:
             logger.info(f"{idx}: {rel} - {e}")
             continue
+
+
+def validate_headers(headers: Iterable[str]) -> None:
+    """Check for data types in the headers"""
+    for header in headers:
+        if ":" in header and header.split(":")[1] not in NEO4J_DATA_TYPES:
```

Comment:
does this consider if it ends with a [] for lists?

Reply:
Good point, it does not currently, I'll add that.

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/110/commits/d3e95ce3ca85dac861edad7676c55409515168e5